bot.py

```diff
@@ -33,6 +34,12 @@ def setup():
                                  'prefix',
                                  'COBALT_key'])
 
+    logger = logging.getLogger('discord')
+    logger.setLevel(logging.DEBUG)
+    handler = logging.FileHandler(filename='logfile.log', encoding='utf-8', mode='w')
+    handler.setFormatter(logging.Formatter('%(asctime)s:%(levelname)s:%(name)s: %(message)s'))
```

Comment:
Could we add the channel it was sent in to the log?

Reply:
This isn't really the server message log it's the discord.py log. I can implement our own logs for the messages in a more view-able format.